doc/elements-tx-format.md

```diff
@@ -0,0 +1,562 @@
+|
+|                                      Output witness #5
+| 00 ................................. Surjection proof length
+| 00 ................................. Range proof length
+```
+
+#### Example 3
+
+Signed pegin claim transaction on liquidv1 (production liquid).
+
+Raw hex:
+```
+0200000001013d0bfb6e61989331d67dac3b6adf067afda20f504cabb1a7364853a68a0bb6df0000004000ffffffff02016d521c38ec1ea15734ae22b7c46064412829c0d0579f0a713d1c04ede979026f0100000000002b09c1001600143921df9000812998ceff99b610437734d552424f016d521c38ec1ea15734ae22b7c46064412829c0d0579f0a713d1c04ede979026f01000000000000002700000000000000000247304402204f84bb59c2af17b76ba71486a3fe4941829f0b5d23bda5c9f49c449489693e5002206e1f3bba907e511e99bf317059264f89a56c50c56fefa82583b8e941b865facc012102e776bdb5d8fc14d24b330c0efbf34227cbba1cf25eb0778aa45f8b7cb34950460608e8092b0000000000206d521c38ec1ea15734ae22b7c46064412829c0d0579f0a713d1c04ede979026f206fe28c0ab6f1b372c1a6a246ae63f74f931e8365e15a089c68d6190000000000160014504b2e2f011741aec712fb2f51dc1537272f284a53020000000113391c88becfe2a855a74fb1890cabef7c81a4be742771046ca0c38e29cfada501000000000000008001e8092b000000000017a9144d733642e6cc020cfd3b2ae52e4ef1b50be6a15f8700000000fd990100004020fe6c0db88a5a5b747b9fa39abbf100641325110dd17c00000000000000000000da21e101b583f398f9d46d97b4d083f6142b0fc67dff71bb2969862fa9e346fe2a05fc627efd0917b8cc42a7bd0900000a874b10883f43cb8d4e8703e8861296688ded81d601a6cb4b68b9680faf7974401da4d3e264de6948f2df86a20cc7dc588a3ff8cc05c27afdd35b1cf50ccf7ae4acf64afb05980884134795302653d442be5f7dd80071ab56b25f57d411545122e475f642f7d98042282c59203e09cd3db6544fc4aac33069ab94057358815a3a6b9e1cb47d349006a33832c6a8a32105d1f16a63982aeea22f19d2c365030d4e3d0bfb6e61989331d67dac3b6adf067afda20f504cabb1a7364853a68a0bb6df9231e9542459531f983b26c6dff457ec9af808191d709c761c22d86a8e59d8200f580dc22957e89c3fe8f3d5dc34f9add75e288041f2b4bc5639ee915535a187c89756e0bd15f3b539864dd4e6dacea4fdd5daa290523e4cf9231cccab094c7efc9f6cf9bc4679908cffcb16a2ff3ec061c66bfa15b886c08c6973776d2fba78035dd70300000000
+```
+
+Deserialization:
+```
+02000000 ............................. Version
+
+01 ................................... Flags (0x01 → witness data is present)
+|
+|                                      Input #1
+| 3d0bfb6e61989331d67dac3b6adf067a
+| fda20f504cabb1a7364853a68a0bb6df ... Outpoint TXID 
+| 00000040 ........................... Output Index (little-endian → 0x40000000 → second most significant bit is 1 → pegin)
+|
+| 00 ................................. ScriptSig length
+| | .................................. ScriptSig (empty: segwit transaction)
+|
+| ffffffff ........................... Sequence number: UINT32_MAX
+
+02 ................................... Num outputs
+|
+|                                      Output #1
+| 01 ................................. Asset header (0x01 → explicit, unblinded value)
+| 6d521c38ec1ea15734ae22b7c4606441
+| 2829c0d0579f0a713d1c04ede979026f ... Key used to generate asset ID
```

Comment:
I suggest commenting that the assetID is `6f02...` here and in the other examples.